Automattic/jetpack
# Google Analytics: initialize via MU-WPCOM plugin

## Proposed changes:
* Initialize Google Analytics package via MU-WPCOM plugin.

### Other information:


## Jetpack product discussion
https://github.com/Automattic/vulcan/issues/312

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:

1. Create/reuse a WoA site. Use Jetpack Beta to checkout this PR's branch (`add/google-analytics-to-mu-wpcom`) for "WordPress.com Features".
2. To activate the MU-WPCOM plugin, go to "Settings -> Hosting Configuration", use SFTP or SSH credentials to add the constant to `htdocs/wp-config.php`: `define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );`
3. In Calypso, go to "Tools -> Marketing -> Traffic" and activate Google Analytics. Enter a measurement ID (e.g. "G-12345"), click "Save".
4. Check the frontend, confirm the tracking code shows up with the correct measurement ID.
5. Deactivate Google Analytics, confirm the tracking code disappears.
6. Use Jetpack Beta to switch Jetpack to the branch `delete/jetpack-module-google-analytics`.
7. Refresh the "Tools -> Marketing -> Traffic" page, confirm the GA card is still there, still deactivated/folded.
8. Repeat steps 3-5, confirm the results are the same.
